### ISSUE TITLE
Add and Start Website Pages

### ISSUE THREAD

**Issue by JJ27** [750B/750B.github.io#9]
index.html links to About Us, Work During Pandemic, etc. 

Those pages should at least be created so the links go somewhere. They should also have similar styling to the main page. Gotta keep that consistent brand 😏.

**Comment by NicholasS13** [comment]
Alright will add once new PR is approved (modified tab names)
